### DISCUSSION TITLE
Define device by MAC address?

### DISCUSSION THREAD

**Discussion by gauzilla** [sbidy/wiz_light#161] [Q&A]
Is there any way to define the bulbs via MAC address instead of IP?

**Comment by cwross** [comment]
The IP would be required regardless so Home Assistant can establish a connection with the bulb. But this brings up a good point. I tried to edit my Wiz light entities and discovered that it is not possible to do so from the UI because a unique identifier can't be set. See screenshot. Anyway for UUID functionality to be added?
![uuids](https://user-images.githubusercontent.com/15680744/127758793-f7c26bec-8a1a-4a74-977d-a3f51aede3d6.png)
